pyfakefs/fake_filesystem.py

```diff
@@ -468,24 +497,24 @@ def size(self, st_size):
         self.epoch += 1
 
     @property
-    def path(self):
+    def path(self) -> Union[str, bytes]:
```

Comment:
Yes, that would definitely be better - I had a hard time with these, and IIRC I changed it because it caused a lot of errors. But I admit that I didn't understand it, will have another go.

Reply:
Changed this meanwhile as proposed.